main.js

```diff
@@ -4,10 +4,24 @@
  **/
 'use strict'
 
-const { Rewriter } = require('node-gyp-build')(__dirname)
 const { getPrepareStackTrace } = require('./js/stack-trace/')
 
+class DummyRewriter {
+  rewrite (code, file) {
+    return code
+  }
+}
+
+function getRewriter () {
+  try {
+    return require('node-gyp-build')(__dirname).Rewriter
+  } catch (e) {
+    // do nothing
```

Comment:
Maybe it would be more specific if we return `DummyRewriter` here instead of in line 21.